apps/site/pages/docs/go-live/overview.mdx

```diff
@@ -0,0 +1,223 @@
+| :--------- | :----------------------------------------------------------------- |
+| Agency and VTEX Support    | [Integrating the VTEX Checkout](/docs/go-live/4-integrating-the-vtex-checkout)  |
+</details>
+
+
+<b><CheckboxField label="Integrating the VTEX Order Placed and My Account" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>Clone, edit, and install the <b><a href="https://github.com/vtex/faststore-vtex-integrations/">faststore-vtex-integrations</a></b> app on your VTEX account.</summary>
+| Management | Reference                                                          |
+| :--------- | :----------------------------------------------------------------- |
+| Agency and VTEX Support    | [Integrating the VTEX Order Placed and My Account](/docs/go-live/5-integrating-the-vtex-order-placed-and-my-account)  |
+</details>
+
+
+<b><CheckboxField label="Checking the pages in the secure domain" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>Test your website links to pages being rendered in the secure subdomain, such as the Login, Checkout, My Account, and Order Placed pages.</summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency     | Coming soon!  |
+</details>
+
+---
+
+### Headless Customization
+
+<b><CheckboxField label="Integrating the storefront with VTEX Headless CMS" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>Set up the preview flow between the storefront and the Headless CMS</summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency     | [VTEX Headless CMS]()  |
+</details>
+
+---
+
+### SSL Configuration
+
+<b><CheckboxField label="Checking the pages SSL certificate" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>Check if all pages of your store have the correct SSL certificate configured.</summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency     | Coming soon!  |
+</details>
+
+---
+
+### Testing and performance
+
+<b><CheckboxField label="Performing Lighthouse audits" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>Run Lighthouse audits on your store's pages to guarantee you are going live with great performance</summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency     | Coming soon!  |
+
+- Run Lighthouse audits on your store's Home Page, Product Listing Page (PLP), Product Details Page (PDP), and custom pages.
+- All pages must have Lighthouse Score equal or more than 60 on mobile devices.
+- Any publication will be automatically blocked by the system until all checks pass.
+</details>
+
+
+<b><CheckboxField label="Sentry" id="checkboxfield-default" /></b>       
+
+<details>
+<summary></summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency and VTEX Support    | Coming soon!  |
+
+</details>
+
+
+<b><CheckboxField label="Setting up A/B testing" id="checkboxfield-default" /></b>       
```

Comment:
The correct name for this should be Traffic Split. A/B Testing is a feature that will be enabled by planner in the future.